internal/api/installation_test.go

```diff
@@ -525,8 +525,18 @@ func TestUpdateInstallation(t *testing.T) {
 	require.NoError(t, err)
 
 	ensureInstallationMatchesRequest := func(t *testing.T, installation *model.Installation, request *model.PatchInstallationRequest) {
-		require.Equal(t, *request.Version, installation.Version)
-		require.Equal(t, *request.License, installation.License)
+		if request.Version != nil {
```

Comment:
Test should still pass if `request.X == nil` and `installation.X` does not?

Reply:
Yeah, the request logic was moved to pointer patching a while back so nil means there would be no change to the underlying installation. It also means that we can't check that `request.VALUE` == `installation.VALUE` if the first in nil so this check is definitely needed.

The only reason tests were passing up till now is that every subtest was passing in all the patch values that could be nil. My new test with an empty patch payload was how I uncovered that we should be doing the if logic here.